packages/marketing-analytics-browser/generated/amplitude-gtm-snippet.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
```

Comment:
Since ci will auto-create this file based on the latest SDK, I will delete this file before merging. This is current proof of if the logic has been generated correctly.

Reply:
Delete this cause build error, will keep this.